packages/config/tailwind/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require("./tailwind.config.js");
```

Comment:
Look into the entrypoints of this package for ease-of-imports

- have the tailwind-config in index.js and not just act as a barrel file?
- rename postcss.config.js -> postcss so the import would simply be "@agreeto/tailwind-config/postcss"